repose-aggregator/components/filters/rackspace-identity-basic-auth/src/main/scala/org/openrepose/filters/rackspaceidentitybasicauth/RackspaceIdentityBasicAuthHandler.scala

```diff
@@ -1,11 +1,13 @@
 package org.openrepose.filters.rackspaceidentitybasicauth
```

Comment:
So, yeah. not strictly required in scala, it appears all statement are part of the case until the next one, so the only place they are probably necessary are when nesting, which is pretty gross and should probably be extracted into a method anyway